### PR TITLE
[FIX] project, rating: project percentage satisfaction compute performance

### PR DESCRIPTION
**Pull request by beledouxdenis**
This is basically a backport of revision
47479401cb3f685788d76abe01d3d2bc21ff11e3
to gain the performance on the project
percentage satisaction computation

but without introducing the new parent mixin,
which would not be a stable change.

### CODE REVIEW

File: addons/rating/models/rating.py [hunk 1]
```diff
@@ -347,3 +349,31 @@ def rating_get_stats(self, domain=None):
         for rate in data['repartition']:
             result['percent'][rate] = (data['repartition'][rate] * 100) / data['total'] if data['total'] > 0 else 0
         return result
+
+    @api.multi
```
Comment (jem-odoo) [inline addons/rating/models/rating.py]: Rather `api.model` as we are not using `self.ids` (to avoid confusion).

**Comment by jem-odoo** [comment]
Small comment, otherwise: LGTMartine !

**Comment by beledouxdenis** [comment]
@robodoo r+

**Comment by beledouxdenis** [comment]
@robodoo retry

**Comment by beledouxdenis** [comment]
@robodoo r+

**Comment by beledouxdenis** [comment]
@robodoo merge la alors gros tarba

**Comment by beledouxdenis** [comment]
@robodoo rebase-ff

**Pull request closed without merging**